common/utils/ga.js

```diff
@@ -17,7 +17,10 @@ export function trackIfOutboundLink(url: string) {
 export type GaEvent = {|
   category: string,
   action: string,
-  label: string
+  label?: string,
+  value?: string | number,
```

Comment:
is `value` allowed to be a string for GA? I thought I maybe read somewhere that it _had_ to be a number. Not sure though.

Reply:
https://stackoverflow.com/a/26250714

Reply:
you're right